tests/atoms/gaia_apps.js

```diff
@@ -372,12 +372,31 @@ var GaiaApps = {
     };
     return result;
   },
+  /**
+  * Install the app with the specified ManifestURL
+  */
+  installWithManifestURL: function(manifestURL) {
```

Comment:
Side note: I checked the content of this file, and there is no other way to install or uninstall an app. Giving the manifestUrl is the only way.

Hence, I wonder if giving this precision in the function name is worth it. How about some simpler as `install` and `uninstall`?